### PR TITLE
lps-89518

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties [hunk 1]
```diff
@@ -58,7 +58,7 @@
 #if (${liferayVersion.startsWith("7.0")})
     liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.0.6-ga7/liferay-ce-portal-tomcat-7.0-ga7-20180507111753223.zip
 #elseif (${liferayVersion.startsWith("7.1")})
-    liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.1.1-ga2/liferay-ce-portal-tomcat-7.1.1-ga2-20181112144637000.tar.gz
+    liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.1.2-ga3/liferay-ce-portal-tomcat-7.1.2-ga3-20190107144105508.tar.gz
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties:61]: Can you run `installCache updateFileVersions` for project-templates subdir and then see if all the tests pass?

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Can you bump the project-template-tokens for portal.tools.bundle.support?  thanks

**Comment by gamerson** [comment]
can you rebase and resend?  thanks

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/petershin/liferay-portal/pull/393. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**